datasets/m_lama/README.md

```diff
@@ -0,0 +1,281 @@
+    - [Social Impact of Dataset](#social-impact-of-dataset)
+    - [Discussion of Biases](#discussion-of-biases)
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+    - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:**
+http://cistern.cis.lmu.de/mlama/
+- **Repository:**
+https://github.com/norakassner/mlama
+- **Paper:**
+@article{kassner2021multilingual,
+  author    = {Nora Kassner and
+               Philipp Dufter and
+               Hinrich Sch{\"{u}}tze},
+  title     = {Multilingual {LAMA:} Investigating Knowledge in Multilingual Pretrained
+               Language Models},
+  journal   = {CoRR},
+  volume    = {abs/2102.00894},
+  year      = {2021},
+  url       = {https://arxiv.org/abs/2102.00894},
+  archivePrefix = {arXiv},
+  eprint    = {2102.00894},
+  timestamp = {Tue, 09 Feb 2021 13:35:56 +0100},
+  biburl    = {https://dblp.org/rec/journals/corr/abs-2102-00894.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org},
+  note      = {to appear in EACL2021}
+}
+
+@inproceedings{petroni2019language,
+  title={Language Models as Knowledge Bases?},
+  author={F. Petroni, T. Rockt{\"{a}}schel, A. H. Miller, P. Lewis, A. Bakhtin, Y. Wu and S. Riedel},
+  booktitle={In: Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing (EMNLP), 2019},
+  year={2019}
+}
+- **Leaderboard:**
+- **Point of Contact:**
+http://cistern.cis.lmu.de/mlama/
+
+
+
+### Dataset Summary
+
+This dataset provides the data for mLAMA, a multilingual version of LAMA. 
+Regarding LAMA see https://github.com/facebookresearch/LAMA. For mLAMA
+the TREx and GoogleRE part of LAMA was considered and machine translated using 
+Google Translate, and the Wikidata and Google Knowledge Graph API. The machine
+translated templates were checked for validity, i.e., whether they contain 
+exactly one '[X]' and one '[Y]'.
+
+This data can be used for creating fill-in-the-blank queries like 
+"Paris is the capital of [MASK]" across 53 languages. For more details see
+the website http://cistern.cis.lmu.de/mlama/ or the github repo https://github.com/norakassner/mlama.
+
+### Supported Tasks and Leaderboards
+
+Language model knowledge probing.
+
+### Languages
+
+This dataset contains data in 53 languages: 
+af,ar,az,be,bg,bn,ca,ceb,cs,cy,da,de,el,en,es,et,eu,fa,fi,fr,ga,gl,he,hi,hr,hu,hy,id,it,ja,ka,ko,la,lt,lv,ms,nl,pl,pt,ro,ru,sk,sl,sq,sr,sv,ta,th,tr,uk,ur,vi,zh
+
+## Dataset Structure
+For each of the 53 languages and each of the 43 relations/predicates there is a set of triples.
+
+### Data Instances
+
+Each triple consists of an object, a predicate and a subject. For each predicate there is a template available. 
+
```

Comment:
Could you provide one example from the dataset please here ?

To do so feel free to use `load_dataset` and take the first sample `dataset["train"][0]`

If it's too big you can crop it and only show the first fields, and add "..." at the end.

Reply:
Good idea, I added one example.